### PR TITLE
Setup instructions

### PR DESCRIPTION
**Pull request by michaelmcinerney**
Instructions mainly for macOS, and a small update for Python

### CODE REVIEW

File: docs/setup.md [hunk 1]
```diff
@@ -49,13 +49,42 @@ sudo apt-get install \
...
+brew tap messense/macos-cross-toolchains
+brew install x86_64-unknown-linux-gnu aarch64-unknown-linux-gnu
+
+brew tap riscv/riscv
+brew install riscv-tools
+```
+
+Note that CMake will require the x86 compiler before it can be invoked.
+
+The instructions in the sections below should apply for macOS.
```
Comment (lsf37) [inline docs/setup.md:77]:
```suggestion
The instructions in the sections below should apply for both, Linux and macOS.
```

File: docs/setup.md [hunk 2]
```diff
@@ -49,13 +49,42 @@ sudo apt-get install \
     mlton-compiler haskell-stack
 ```
 
+### macOS packages
+
+First install Homebrew from [their website][homebrewwebsite]. Most of the dependencies
+can be installed with
+
+``` brew install git libxml2 ncurses librsvg dtc cmake ninja texlive rysnc python ccache zstd
+```
```
Comment (lsf37) [inline docs/setup.md:58]: the command needs to be on a separate line from the code fence (```)

`haskell-stack` seems to be missing from the list, which brew also has.

Comment (michaelmcinerney) [reply docs/setup.md:58]: Should I include `haskell-stack` in the list here, or should I mention in the Haskell section that macOS users can just perform `brew install haskell-stack`?

File: docs/setup.md (hunk 1 above)
Comment (michaelmcinerney) [reply docs/setup.md:77]: Actually, I'd prefer it without the comma, if you don't mind

File: docs/setup.md (hunk 2 above)
Comment (lsf37) [reply docs/setup.md:58]: include it in the package list (same as for Linux)

File: docs/setup.md (hunk 1 above)
Comment (lsf37) [reply docs/setup.md:77]: Happy to leave the comma up to you :-)

**Pull request merged**